persistence/src/main/java/com/softserve/osbb/model/Region.java

```diff
@@ -49,4 +56,14 @@ public void setName(String name) {
 		this.name = name;
 	}
 
+	@OneToMany(mappedBy="region")
+	@JsonIgnore
+	public Collection<City> getCities() {
+		return cities;
+	}
+
+	public void setCities(Collection<City> cities) {
+		this.cities = cities;
+	}
+	
```

Comment:
Line 15:  import javax.persistence.*;  is redudant after lines 10-12 ( import javax.persistence.GenerationType;	import javax.persistence.Id; import javax.persistence.Table;)
should be removed

Line 59: fetchType = LAZY  should be used to prevent resources overload